packages/ember/src/components/nrg/mktg/header.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+
+export default class Header extends Component {
+  get titleClassList() {
+    let classes = ['col', 'd-flex', 'justify-content-center', 'flex-row'];
```

Comment:
If you'd rather keep these as one string joined with spaces, I'd be okay with that too
```js
let classes = ['col d-flex justify-content-center flex-row'];
```

Reply:
We only need to be careful when we don't know what (if anything) the user is going to pass in. Although I think going to yielded components vs blocks will solve that problem anyway.

Reply:
A new thought - converting this to a Glimmer JS/TS component will let us use the best of both worlds here! Since you can define "subcomponents" without them needing their own files, you can use both named yield blocks and yielded components so that attributes can still be passed around. If that doesn't make sense, I'm happy to work with you to come up with an example.